openedx/core/djangoapps/programs/tests/test_backpopulate_program_credentials.py

```diff
@@ -90,7 +77,7 @@ def test_handle(self, commit, mock_task):
         call_command('backpopulate_program_credentials', commit=commit)
 
         if commit:
-            mock_task.assert_called_once_with(self.alice.username)
+            mock_task.assert_called_with(self.alice.username)
```

Comment:
The certificates are generated above. Previously these tests were using the enable certification flag I asked you above and setting it to false. With that flag, the backpopulate_program_credentials was the only thing awarding certificates. Without it, the certificate creation also triggers the signal to award certificates. I modified some of the tests to try to still test the thing they were trying to test by checking call count or that the other username was not called.